src/js/components/main/board.js

```diff
@@ -0,0 +1,293 @@
+export const keyCodes = {
+  ru: [
+    'ё',
+    '1',
+    '2',
+    '3',
+    '4',
+    '5',
+    '6',
+    '7',
+    '8',
+    '9',
+    '0',
+    '-',
+    '=',
+    'Backspace',
+    'Tab',
+    'й',
+    'ц',
+    'у',
+    'к',
+    'е',
+    'н',
+    'г',
+    'ш',
+    'щ',
+    'з',
+    'х',
+    'ъ',
+    '\\',
+    'Del',
+    'CapsLock',
+    'ф',
+    'ы',
+    'в',
+    'а',
+    'п',
+    'р',
+    'о',
+    'л',
+    'д',
+    'ж',
+    'э',
+    'Enter',
+    'Shift',
+    'я',
+    'ч',
+    'с',
+    'м',
+    'и',
+    'т',
+    'ь',
+    'б',
+    'ю',
+    '.',
+    '↑',
+    'Shift',
+    'Ctrl',
+    'Win',
+    'Alt',
+    'Space',
+    'Alt',
+    '←',
+    '↓',
+    '→',
+    'Ctrl',
+  ],
+  en: [
+    '`',
+    '1',
+    '2',
+    '3',
+    '4',
+    '5',
+    '6',
+    '7',
+    '8',
+    '9',
+    '0',
+    '-',
+    '=',
+    'Backspace',
+    'Tab',
+    'q',
+    'w',
+    'e',
+    'r',
+    't',
+    'y',
+    'u',
+    'i',
+    'o',
+    'p',
+    '[',
+    ']',
+    '\\',
+    'Del',
+    'CapsLock',
+    'a',
+    's',
+    'd',
+    'f',
+    'g',
+    'h',
+    'j',
+    'k',
+    'l',
+    ';',
+    "'",
+    'Enter',
+    'Shift',
+    'z',
+    'x',
+    'c',
+    'v',
+    'b',
+    'n',
+    'm',
+    ',',
+    '.',
+    '/',
+    '↑',
+    'Shift',
+    'Ctrl',
+    'Win',
+    'Alt',
+    'Space',
+    'Alt',
+    '←',
+    '↓',
+    '→',
+    'Ctrl',
+  ],
+  keycodeEn: [
```

Comment:
Please follow the camelCase convention: keyCodeEn/keyCodeRu. Also, the whole constant is keyCodes, then we have en, ru and again keyCodeEn, keyCodeRu. Please describe what are those specifically. Maybe we can come with better naming here.